### ISSUE TITLE
namesdb app throws 500

### ISSUE THREAD

**Issue by GeoffFroh** [densho/ddr-public#130]
`https://ddr.densho.org/names` is throwing a 500 error. The app on stage is showing a transport error that suggests the namesdb app is attempting to use the ES index for the `ddr-public` app data, rather than the `namesdb` data.

```
TransportError(404, '{"_index":"ddrpublic-20180713a","_type":"_all","_id":"names","found":false}')
```
see: https://github.com/densho/namesdb/issues/4

**Comment by gjost** [comment]
This is a problem in the `ddr-public` site not with the `namesdb` app.

**Comment by gjost** [comment]
Oops thought this was the bug filed in namesdb.  My bad.

**Comment by gjost** [comment]
Namesdb is an app with the ddrpublic site.  It is included in `ddrpublic/ui/urls.py`:
```
    url(r'^names', include('names.urls')),
```
The inclusion is missing a trailing slash.  This URL works OK "https://ddr.densho.org/names" but this one did not "https://ddr.densho.org/names/".

The problem was that when the trailing slash was added it no longer matched and the URL was picked up by the catch-all object detail view:
```
    url(r'^(?P<oid>[\w\d-]+)/$', objects.detail, name='ui-object-detail'),
```

**Comment by gjost** [comment]
Fixed in ddr-public commit af7bca2 for package ddrpublic-master_3.2.15~deb9.
